manager/gui/src/components/alerts/LrAlert.vue

```diff
@@ -70,25 +73,31 @@ import {
   PaginationModel,
 } from "@/lib/components/SgtDataTable/SgtDataTableFilterSortPag.model";
 import { SgtFilterObject } from "@/lib/components/SgtChips/SgtFilterObject.model";
+import SgtDialog from "@/lib/components/SgtDialog/SgtDialog.vue";
+import { SgtDialogModel } from "@/lib/components/SgtDialog/SgtDialog.model";
+import { create } from "vue-modal-dialogs";
+
 @Component({
   name: "LrAlert",
   components: { SgtDataTable, LrAlertDialog, LrAlertComments },
 })
 export default class LrAlert extends Vue {
-  @Prop({ required: false, default: "" }) private severity: string;
   @Prop({ required: false }) private alertId: string;
+  @Prop({ required: false }) private severity: string;
   alertConst: any = JSON.parse(JSON.stringify(lrAlertConst));
   alerts: any = [];
   showDataTable = false;
   chips: SgtFilterObject[] = [];
   showAlertDetailsDialog = false;
   selectedRecord: any = null;
   showAlertCommentsDialog = false;
+  public acknowledgeModal = create<SgtDialogModel>(SgtDialog);
 
   mounted() {
-    Api.getData("alerts/list", { isDummy: true }).then((resp: any) => {
-      this.alerts = resp["list"];
-    });
+    this.getAlertsList();
```

Comment:
this should be in else part, since in case of severity it is making 2 api calls

Reply:
Moved the getAlertsList function call to the else part.